__tests__/app.test.js

```diff
@@ -302,6 +302,26 @@ describe('GET /api/users', () => {
         })
     })
 }) 
+describe('QUERY = topic GET /api/articles', () => {
+    it('returns 200 and returns articles of specific topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({body}) => {
+            body.articles.forEach((article) => {
```

Comment:
remember to do a length check if you're going to forEach through something in your test.